### PR TITLE
Added: about page

### PR DESCRIPTION
**Pull request by smriteemanandhar**
Request to merge about page in production

### CODE REVIEW

File: src/components/Card.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'bootstrap/dist/js/bootstrap.bundle.min.js';
+
+const Card = () => {
+  return (
+    <div>
+        <div className="card" style={{width: "20rem"}}>
```
Comment (sanjayguwaju) [inline src/components/Card.jsx]: user col-3 or col-6 for the better responsive bootstrap code

File: src/components/Card.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react'
```
Comment (sanjayguwaju) [inline src/components/Card.jsx]: We don't need to add this line of code in in React 18.2

**Review by sanjayguwaju — CHANGES_REQUESTED** [review]

File: src/components/Card.jsx (hunk 1 above)
Comment (smriteemanandhar) [reply src/components/Card.jsx]: Its not working when i use col class.

File: src/components/Card.jsx (hunk 2 above)
Comment (smriteemanandhar) [reply src/components/Card.jsx]: Done

**Review by sanjayguwaju — APPROVED** [review]

**Pull request merged**